src/Executables/CombineH5/CombineH5.cpp

```diff
@@ -51,7 +55,9 @@ void combine_h5(const std::string& file_prefix, const std::string& subfile_name,
                 const std::string& output) {
   // Parses for and stores all input files to be looped over
   const std::vector<std::string>& file_names =
-      file_system::glob(file_prefix + "*.h5");
+      file_system::glob(file_prefix + "[0-9]*.h5");
+  Parallel::printf("Processing files:\n%s\n",
+                   std::string{MakeString{} << file_names}.c_str());
```

Comment:
I think this is ok because we'll be replacing it with a CLI soon where the user can determine how much they want printed.

Reply:
Yea, @aasthabagree21 is going to clean this up a bit more for the CLI :)